static/app/views/organizationGroupDetails/groupMerged/mergedToolbar.tsx

```diff
@@ -115,6 +116,17 @@ class MergedToolbar extends React.Component<Props, State> {
             </Button>
           </Confirm>
 
+          <Confirm
+            onConfirm={onSplit}
+            message={t(
+              'These events will be grouped into a new issue by more specific criteria (for instance more frames). Are you sure you want to split them out of the existing issue?'
+            )}
+          >
+            <Button size="small" title={t(`Splitting out ${unmergeCount} events`)}>
```

Comment:
```suggestion
            <Button size="small" title={tct('Splitting out [unmergeCount] events', {unmergeCount})}>
```

`tct` can better help translators 😉